omnibus/files/server-ctl-cookbooks/infra-server/recipes/fix_permissions.rb

```diff
@@ -24,3 +24,18 @@
 execute "find #{GEM_PATH} -perm /u=r,g=r,o=r ! -perm /u=x -exec chmod 644 {} \\;" do
   user 'root'
 end
+
+# We backup the files and templates in the folder /var/opt/opscode/local-mode-cache/backup
+# the default vaule for the file and template resource is 5 i.e, there will be upto 5 backups of the
+# files and templates that we use in the server-ctl cookbook. This includes the configs files also.
+# To stop taking the backup, we need to explicitly mention the property backup as 'false' in all the
+# usage of file and template resource in the server-ctl cookbook.
+# We can secure the backups by changing the permission of /var/opt/opscode/local-mode-cache/backup
+# to root user read only. (CVE-2023-28864)
+
+directory '/var/opt/opscode/local-mode-cache/backup' do
```

Comment:
Would this mean that/:
1. For current install, after this script executes there will be no backups written to this folder (path specified by user via options)
2. For new installs, no backups will ever get saved to this folder (path specified by user in options)

Reply:
Could we use the consts from CHEF::Utils for the path. so that this changes does not break cinc?